### DISCUSSION TITLE
Data download link doesn't work

### DISCUSSION THREAD

**Discussion by DinhGiaHuyLe** [matsim-melbourne/demand#28] [General]
Hi,
I tried to use this link but it can't be accessed (https://cloudstor.aarnet.edu.au/plus/s/vILZLoFiqKU58Sp). Can anyone help me check this?

Regards,
Huy

**Comment by jafshin** [comment]
Thank you Huy for your question and apologies that the link in the master branch is not working, the updated readme is in the dev branch, with a working link to the data repositories: https://github.com/matsim-melbourne/demand/tree/dev/data , we will update the master branch soon as well.
